tests/00-check-examples.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source "${dir}/helpers.bash"
+# dir might have been overwritten by helpers.bash
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+TEST_NAME=$(get_filename_without_extension $0)
+LOGS_DIR="${dir}/cilium-files/${TEST_NAME}/logs"
+redirect_debug_logs ${LOGS_DIR}
+
+set -ex
+
+for p in ../examples/policies/*.json ../examples/demo/*.json ; do
+ if ! cilium policy validate $p; then
```

Comment:
This won't be enough, please create a GH issue to track the enforcement of the policy, not only the validation.

Reply:
Created #1912, please add to it if something is missing.